hacks_mbof/urls.py

```diff
@@ -43,7 +45,20 @@
     url(r'^bower_components/(?P<path>.*)$', serve, {
         'document_root': 'mbofui/bower_components',
     }),
+]
+
+if 'djangosaml2' in settings.INSTALLED_APPS:
+    urlpatterns += (
+        url(r'^accounts/', include('djangosaml2.urls')),
+        url(r'^test/', 'djangosaml2.views.echo_attributes'),
+    )
```

Comment:
I figured it out by writing this little script:

l = [1,2,3]
print l
l += (4,)
print l

The key is the ',' after the 4. It appears that any time there is a comma after an entry, python will interpret the entry as a list. The same code will not work when the comma is missing. So it would seem:

l += (4,) is the same as l += [4]